libraries/AP_Relay/AP_Relay_Params.cpp

```diff
@@ -0,0 +1,32 @@
+#include <AP_Common/AP_Common.h>
+#include "AP_Relay_Params.h"
+
+const AP_Param::GroupInfo AP_Relay_Params::var_info[] = {
+    // @Param: FUNCTION
+    // @DisplayName: Relay function
+    // @Description: The function the relay channel is mapped to.
+    // @Values: 0:None,1:Relay 1,2:Relay 2,3:Relay 3,4:Relay 4,5:Relay 5,6:Relay 6,7:Ignition,8:Starter
+    // @User: Standard
+    AP_GROUPINFO_FLAGS("FUNCTION", 2, AP_Relay_Params, function, (float)Function::none, AP_PARAM_FLAG_ENABLE),
```

Comment:
maybe make it 1,2,3 for index

Reply:
maybe RELAY1_TYPE instead of FUNCTION with values 0=None,1=SimpleRelay,2=Ignition,3=Starter
this removes the indirections, so relay1 is relay number 1

Reply:
I agree but we could keep the param name as RELAY1_FUNCTION is Michael wants it like it is.

Reply:
Done, I kept the `FUNCTION` just because I liked the symmetry with servo's, and it seems clear to me what it does.